modules/sns-cronjob/main.tf

```diff
@@ -0,0 +1,43 @@
+terraform {
+    required_providers{
+        aws = {
+            source = "hashicorp/aws"
+            version = "~> 3.5.0"
+        }
+    }
+}
+
+provider "aws" {
+  region = "us-east-1"
+}
+
+#SNS TOPIC creates SNS Topic resources on AWS
+resource "aws_sns_topic" "default" {
+  count = var.enabled && var.enable_topic ? 1 : 0
```

Comment:
why we have two enable controlling variables? in case if disabled we maybe will have  issue in resources where we use this resource outputs